usr/share/rear/skel/default/etc/scripts/system-setup

```diff
@@ -116,9 +116,10 @@ if test -s "/md5sums.txt" ; then
             # In debug mode let the user confirm to proceed:
             read -p "Press ENTER to proceed 'bona fide' nevertheless "
         else
-            # In non-debug mode wait 10 seconds so that the user can read the md5sum output:
+            # In non-debug mode wait 10 seconds so that the user can read the md5sum output
+            # unless in unattended_recovery mode where there is no user who reads something:
             echo -e "Proceeding 'bona fide' nevertheless...\n"
-            sleep 10
+            unattended_recovery || sleep 10
```

Comment:
❤️ faster is better

Reply:
Yes, I think it is a main feature of ReaR
that "rear recover" is very fast.
What ReaR itself does (primarily disk layout setup)
takes almost no time compared to "third-party" tasks like
restoring the backup (usually takes very most time) and
recreating the system's initrd (second most time) and
reinstalling the bootloader (third most time).